koku/api/resource_types/test/Aws/aws_accounts/tests_views.py

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
The structure of the test directory should match the structure of the `/resources_types` directory. We don't have provider specific folders (i.e. Aws, Azure, Gcp) in the `/resources_types` dir, so we shouldn't have them here either.

Reply:
This has been fixed!